samples/system-test/dlp_delete_job.test.js

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
`s/system-test/test/g`

We should now have tests in `./samples/test` and deprecate the phrase `system-test` for our sample tests.

( I think? ) => this is a [refactor](https://github.com/googleapis/nodejs-translate/pull/237/files#diff-65a97c36fccefc69ea92b229617dab15) that @JustinBeckwith did today

Reply:
@jmdobry

Reply:
( fwiw – I think `./samples/test` is clean. and people understand it. it's idiomatic. )
•
( when I read "system-test" my brain takes a moment to wonder "what types of tests are these?" )
•
( i would support this 🙈 )

Reply:
Yeah, I tend to just switch these to `test` as I go.